src/main/java/ca/team2706/frc/robot/logging/Logging.java

```diff
@@ -0,0 +1,60 @@
+package ca.team2706.frc.robot.logging;
+
+import ca.team2706.frc.robot.Robot;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.config.Configurator;
+
+public class Logging {
```

Comment:
This whole class should probably get some javadoc descriptions for the class as well as all the methods inside of it.